### DISCUSSION TITLE
Marking External Assets Not Working?

### DISCUSSION THREAD

**Discussion by Dillpickleschmidt** [Gorgious56/asset_browser_utilities#108] [Q&A]
Hey, I'm new to using the asset browser in general, so maybe I'm missing something really basic...
I have a bunch of individual blender files each with a unique asset, and I want to add all of them to an asset library so I can pull any of them into any scene I'm working on. If I try to batch mark any external files, Blender will open up all of the files/folders I specify and go through each very quickly, but when it's done and says "Work completed !" I have an empty asset browser. Doing this for assets in the current file seems to work fine, they're added to the library. I only installed and enabled the addon, hit batch operator, clicked external folders, and navigated to my folder containing all of the blend files. It seems to be filtering for objects only by default, which is what I want. Have I missed any steps?

**Comment by Gorgious56** [comment]
Hello ! By default the asset browser only displays the assets in the current file. You have to setup the folder for your asset library in Edit > Preferences > File Paths and click on the + icon in the lower right of the "Asset Libraries" subpanel. Then after it's setup go to your asset browser, and change "Current File" to the name of your library.

![image](https://user-images.githubusercontent.com/25156105/200419493-bbb8dd55-3389-4425-822d-9bc6991c7c86.png)


Hope that sorted it for you :)
